docs/source/reference/release_notes.rst

```diff
@@ -2,6 +2,37 @@
  Release History
 =================
 
+1.6.2 (2021-08-30)
+==================
+
+Fixes
+-----
+
+* Prevent set_and_wait from accepting a failed read when tolerances are set.
+* Added missing _metadata_keys and kwargs to FakeEpicsSignal, to better 
+  mimick EpicsSignal.
+* EpicsMotor will now restore the ``SET`` field back to 0 if an error occurs
+  during ``set_current_position``.
+* ``SequenceId`` PV on EigerDetectorCam is now correctly read and write.
```

Comment:
Passive present form?
```suggestion
* ``SequenceId`` PV on EigerDetectorCam is now correctly read and written.
```

Reply:
`read and write` in the sense of not `read-only`, but I can see how it may be confusing... will adjust wording a bit.